webapp/src/components/containers/DropdownCard.vue

```diff
@@ -24,14 +24,26 @@
 </template>
 
 <script setup lang="ts">
-import { ref } from "vue";
+import { ref, onMounted } from "vue";
 
 import UpArrow from "@/assets/svg/UpArrow.vue";
 import DownArrow from "@/assets/svg/DownArrow.vue";
 
 import IconCardHeader from "./IconCardHeader.vue";
 
+const props = defineProps({
+  initiallyOpen: {
+    type: String,
+    default: "true", //open by default
+  },
+});
+
 const opened = ref(true);
+
+onMounted(() => {
+  opened.value = JSON.parse(props.initiallyOpen); 
+});
```

Comment:
`onMounted` [is quite special](https://vuejs.org/guide/essentials/lifecycle.html), it doesn't get triggered when data change for example. better to watch for prop to change instead, so that if someone change the prop from the outside, it will behave as expected.

```suggestion
watch(props, ({ initiallyOpen }) => {
  opened.value = initiallyOpen;
});
```